pkg/pillar/dpcreconciler/linuxitems/adapter.go

```diff
@@ -131,13 +131,6 @@ func (c *AdapterConfigurator) Create(ctx context.Context, item depgraph.Item) er
 		// Managed by the wwan microservice, nothing to do here.
 		return nil
 	}
-	// Make sure that the interface is UP.
-	if err := netlink.LinkSetUp(link); err != nil {
```

Comment:
We need to make sure the interface is set UP even when it is not bridge by NIM, i.e. when `c.isAdapterBridgedByNIM(adapter)` returns false. Please add this into `if !c.isAdapterBridgedByNIM(adapter) { ... }`

Reply:
Done